scripts/re-ingest.py

```diff
@@ -0,0 +1,118 @@
+"""
+This script performs a re-ingestion of submissions that are stored in the database
+
+Usage:
+    python script.py [file_path] [endpoint_url]
+
+Arguments:
+    directory_path (str): Path to a file containing submission UUIDs to be re-ingested
+    base_url (str): URL of the where the data-store API is served.
+
+Examples:
+    python script.py /path/to/file http://localhost:8080/
+
+Dependencies:
+    - pandas
+    - requests
+
+Note:
+    - The script assumes that the UUIDS supplied in the file are linked to submissions in the database.
+    - The output will be saved in the directory from which you run this script
+"""
+import argparse
+import json
+from datetime import datetime
+
+import pandas as pd
+import requests
+
+
+def get_file(submission_UUID, endpoint_url):
+    """
+    Post a file to the specified endpoint URL.
+
+    :param endpoint_url: URL of the endpoint to post the file to.
+    :return: Response from the server.
+    """
+    response = requests.get(
+        endpoint_url,
+        params={"submission_UUID": submission_UUID},
+    )
+    return response
+
+
+def post_file(file_bytes, reporting_round, endpoint_url):
+    """
+    Post a file to the specified endpoint URL.
+
+    :param file_path: Path to the file to be posted.
+    :param endpoint_url: URL of the endpoint to post the file to.
+    :return: Response from the server.
+    """
+    response = requests.post(
+        endpoint_url,
+        files={
+            "excel_file": (
+                f"reingest{datetime.now().strftime('%Y%m%dT%H%M%S%z')}",
```

Comment:
Are we happy to overwrite existing file names with this? It will update the `Submission` table to this so we will lose the original file name. Maybe this is fine? Maybe it also makes saving the filename redundant..

Reply:
I've changed this to pull through the file_name stored in the DB. This is hard to test though because in the mock example the file name seems to be: `'C:\\Users\\OShaughnessyC\\Documents\\python\\funding-service-design-post-award-data-store\\tests\\resources\\Post_transform_EXAMPLE_data.xlsx'` but obviously different on every machine

Reply:
oh that's not intended. 

We should be just using the file name rather than the full address. 

You could probably include a small fix in this PR to use `os.path.basename` when we save it to the db

Reply:
I would do this now but it actually seems like Noels lates PR is going to change this example data mock in quite a large way. I'll probably wait and see how this works once the test refactors are fully merged